src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -331,24 +331,493 @@ The following workflows represent some mathematical functions that can be optimi
+| String (default="ml_service-api").
+| `PYTHON_ENTRYPOINT`
+| This entry script starts the service and defines the different functions to deploy the model, score the requests by using the model, and return the results. This specific script is specific to your model. This file should be stored in the Catalog under the `model_as_service_resources` bucket. More information about this file can be found in the <<Customize the Service>> section.
+| String (default="ml_service").
+|===
+
+
+=== Deploy a Model
+
+There is two ways to deploy a model in the started model_server (i.e. where you will deploy, update and call your model) using MLOS:
+
+*Via Swagger UI*
+
+Open the swagger UI.
+
+Open the `get_token` operation and get an api_token by entering your username (default value is `user`).
+
+Open the `deploy` operation and set the provided token and upload the model that need to be deployed.
+
+image::MAS_Deploy_Swagger.png[align=center]
+
+* *via Studio portal*
+
+Open the link:https://try.activeeon.com/Studio[Studio Portal].
+
+Create a new workflow.
+
+Add the `model_as_a_service` bucket.
+
+Drag and drop the `Deploy_Model` task from the bucket.
+
+Execute the Workflow and set the different workflow variables as following:
+
+.Deploy_Model variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `DEPLOY_MODEL_ENDPOINT`
+| the endpoint of the started service.
+| URL
+| `API_EXTENSION`
+| the base path to access the deployment endpoint.
+| String (default="/api/deploy")
+| `MODEL_URL`
+| the URL of the model that need to be deployed.
+| URL
+| `SERVICE_TOKEN`
+| the api token to be able to deploy the model.
+| String
+|===
+
+
+=== Call the service for predictions
+
+There is two ways to call the web service to score input data using MLOS:
+
+*Via Swagger UI*
+
+Open the swagger UI.
+
+Open the get_token` operation and get an api_token by entering your username (default value is `user`).
+
+Open the `predict` operation and set the provided token and the data that you need to score.
+
+image::MAS_Predict_Swagger.png[align=center]
+
+* *via Studio portal*
+
+Open the link:https://try.activeeon.com/Studio[Studio Portal].
+
+Create a new workflow.
+
+Add the `model_as_a_service` bucket.
+
+Drag and drop the `Call_Prediction_Service` task from the bucket.
+
+Execute the Workflow and set the different workflow variables as following:
+
+.Call_Prediction_Service variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `PREDICT_MODEL_ENDPOINT`
+| the endpoint of the started service.
+| URL
+| `PREDICT_EXTENSION`
+| the base path to access the prediction endpoint.
+| String (default="/api/predict")
+| `SERVICE_TOKEN`
+| the api token to be able to deploy the model.
+| String
+| `INPUT_DATA`
+| entry data that needs to be scored by the deployed model.
+| JSON
+| `LABEL_COLUMN`
+| need to be set if data is labeled.
```

Comment:
`need to be set if data is labeled.` => `Name of the label column. Need to be set if data is labeled.`